### ISSUE TITLE
Prevent paste long text (include free spaces)

### ISSUE THREAD

**Issue by parsadoc** [eyoeldefare/textfield_tags#69]
You can paste a long text with many free spaces without any problem! and Input doesn't detect any worlds as tags!
How to prevent that?
Of course, it happens for some texts like this one:

` if(resolution.categories!.any((e) => e.toLowerCase() == tag)) {`

<table>
<tr><td>
<img src="https://user-images.githubusercontent.com/47334201/183579880-065eed7b-2557-4b00-8775-bfb482a7cae1.png" />
</td>
<td>
<img src="https://user-images.githubusercontent.com/47334201/183579895-1010abe2-783d-4341-b06f-7afd7a78eb65.png" />
</td></tr></table>

**Comment by eyoeldefare** [comment]
You're right there. If you paste a text, it won't evaluate the whole text because the widget is listening for only input data entered by the user (as in one text value at a time). If the user pastes a whole text value, it can only check the last of that entered data to see it matches with the conditions of the text separator you gave it.
